examples/high_freq/workflow.py

```diff
@@ -0,0 +1,170 @@
+
+from highfreq_ops import DayFirst, DayLast, FFillNan, Date, Select, IsNull
+
+if __name__ == "__main__":
+
+    # use default data
+    provider_uri = "/nfs_data/qlib_data/yahoo_high_qlib"  # target_dir
+    qlib.init(
+        provider_uri=provider_uri,
+        custom_ops=[DayFirst, DayLast, FFillNan, Date, Select, IsNull],
+        redis_port=-1,
+        region=REG_CN,
+        auto_mount=False,
+    )
+
+    MARKET = "all"
+    BENCHMARK = "SH000300"
+    DROP_LOAD_DATASET = False  # flag wether to test [drop and load dataset]
+
+    # start_time = "2019-01-01 00:00:00"
+    # end_time = "2019-12-31 15:00:00"
+    # train_end_time = "2019-05-31 15:00:00"
+    # test_start_time = "2019-06-01 00:00:00"
+    start_time = "2020-09-14 00:00:00"
+    end_time = "2021-01-18 16:00:00"
+    train_end_time = "2020-11-30 16:00:00"
+    test_start_time = "2020-12-01 00:00:00"
+    ###################################
+    # train model
+    ###################################
+    DATA_HANDLER_CONFIG0 = {
+        "start_time": start_time,
+        "end_time": end_time,
+        "freq": "1min",
+        "fit_start_time": start_time,
+        "fit_end_time": train_end_time,
+        "instruments": MARKET,
+        "infer_processors": [{"class": "HighFreqNorm", "module_path": "highfreq_processor", "kwargs": {}}],
+    }
+    DATA_HANDLER_CONFIG1 = {
+        "start_time": start_time,
+        "end_time": end_time,
+        "freq": "1min",
+        "instruments": MARKET,
+    }
+
+    task = {
+        "dataset": {
+            "class": "DatasetH",
+            "module_path": "qlib.data.dataset",
+            "kwargs": {
+                "handler": {
+                    "class": "HighFreqHandler",
+                    "module_path": "highfreq_handler",
+                    "kwargs": DATA_HANDLER_CONFIG0,
+                },
+                "segments": {
+                    "train": (start_time, train_end_time),
+                    "test": (
+                        test_start_time,
+                        end_time,
+                    ),
+                },
+            },
+        },
+        # You shoud record the data in specific sequence
+        # "record": ['SignalRecord', 'SigAnaRecord', 'PortAnaRecord'],
+        "dataset_backtest": {
+            "class": "DatasetH",
+            "module_path": "qlib.data.dataset",
+            "kwargs": {
+                "handler": {
+                    "class": "HighFreqBacktestHandler",
+                    "module_path": "highfreq_handler",
+                    "kwargs": DATA_HANDLER_CONFIG1,
+                },
+                "segments": {
+                    "train": (start_time, train_end_time),
+                    "test": (
+                        test_start_time,
+                        end_time,
+                    ),
+                },
+            },
+        },
+    }
+    ##=============load the calendar for cache=============
+    # unnecessary, but may accelerate
```

Comment:
Is there a better way to load the cache in advance? @you-n-g

Reply:
I don't have better ideas so far....
We can add some comments to explain that this acceleration doesn't work on windows and mac .